.markdownlint.json

```diff
@@ -2,12 +2,13 @@
   "line_length": {
     "line_length": 120,
     "strict": false,
-    "stern": true,
+    "stern": false,
```

Comment:
The explanation of this is here: https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md013.md

If we don't do this, long links in the TOC will be broken